css/home.css

```diff
@@ -1,4 +1,31 @@
-/* Home page specific styles */
+.footer {
+    background: #22418a;
+    color: white;
+}
+.flex-box {
```

Comment:
This name is not very indicative, in fact it is confusing as it name implies some relation with flex, but it doesn't have such relation.